documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -25,13 +25,34 @@ The Raspberry Pi 5 firmware defaults to loading `kernel_2712.img` because this i
 
 === `arm_64bit`
 
+
 If set to 1, the kernel will be started in 64-bit mode. Setting to 0 selects 32-bit mode.
 
 In 64-bit mode, the firmware will choose an appropriate kernel (e.g. `kernel8.img`), unless there is an explicit `kernel` option defined, in which case that is used instead.
 
-Defaults to 1 on Pi 4s (Pi 4B, Pi 400, CM4 and CM4S), and 0 on all other platforms. However, if the name given in an explicit `kernel` option matches one of the known kernels then `arm_64bit` will be set accordingly.
+Defaults to 1 on Raspberry Pi 4, 400 and Compute Module 4, 4S platforms. Defaults to 0 on all other platforms. However, if the name given in an explicit `kernel` option matches one of the known kernels then `arm_64bit` will be set accordingly.
+
+64-bit kernels come in the following forms:
+
+* uncompressed image files
+* gzip archives of an image
```

Comment:
I wonder if we should remove the plural from these two lines, given that you'll only ever be using one kernel at any given time? And IMHO "archive" usually implies that there's more than one file inside the archive, so perhaps this could be simplified to? (so "uncompressed image file" and "gzip-compressed image file" ?)